README.rst

```diff
@@ -37,6 +37,7 @@ numpy            1.9.0
 nibabel          1.4.0dev
 pydicom          0.9.9
 dcmstack         0.7.0dev
+pymongo          2.8
```

Comment:
Yes, please rebase to latest master and drop this.  That would be appreciated.  Thank you for offering.  I didn't initially ask you do this, because I didn't want to just plop a task on your plate without consulting with you.  

The latest scitran/scitran master should work for you. The latest scitran/scitran changed some naming conventions, causing incompatibility with previously installed scitran docker containers.  You can remove any of the old scitran docker stuff, containers and images.